src/assets/illustrations/052-bike.svg

```diff
@@ -0,0 +1,19 @@
+<svg viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The illustration SVGs should be consistent, which means that they should all have the same `viewbox`. Currently there are different variants (e.g. `viewBox="0 0 80 80"` and `viewBox="0 0 144 144"`).

How I would import the illustrations:
1. Delete all existing illustrations in the project
2. Go to the illustrations overview in Figma: https://www.figma.com/file/FWlI63N28LP7BxqNqj1bGw/sy-004-(Design!)?node-id=5310%3A20878&t=OfCV4q3sqkTirQzm-4
3. Select all illustrations at once and export them (s. screenshot)
4. Remove `width="144"` and `height="144"` in the first line of each svg file
5. Adjust `illustration-files.tsx`
6. Add the illustrations to the page

<img width="1552" alt="Bildschirm­foto 2023-03-02 um 17 08 31" src="https://user-images.githubusercontent.com/57712895/222484950-66e79299-3932-4fa2-b399-7a6f17a2e255.png">

Reply:
When I looked before the pull request, the component library hadn't been updated yet, so I used the ones in the page designs. Of course, it is much more convenient to export all the icons at once.